packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -161,22 +161,32 @@ Follows the [Column Type Philosophy](/packages/nimble-components/src/table/specs
+    -   `'decimal'` - format all values as decimal values (e.g. 123.45) with a number of digits after the separator determined by `decimal-digits` or `decimal-maximum-digits`. Exponential notation is never used. If required, values will be rounded to reach the specified number of decimial digits. Configuring `decimal-digits` to `0` will round the value to the nearest integer and display it with no decimal places. Will be displayed right-aligned by default.
     -   This could be extended to other pre-configured formats in future. Their configuration attributes would be prefixed with the name of the format mode.
     -   **Note:** all of the above will be implemented using a `Intl.NumberFormat` formatter. Nimble will configure the formatter with defaults to match the [visual design spec](https://github.com/ni/nimble/issues/887). The exception is that we will set `useGrouping: true` to achieve `1,000` rather than `1000` because this styles the values in a way that is more human readable.
--   `decimal-digits` - when format is `decimal`, a number that controls how many digits are shown to the right of the decimal separator. Defaults to 2 if unspecified. Formats other than `decimal` ignore `decimal-digits`.
+-   `decimal-digits` - when format is `decimal`, a number that controls how many digits are shown to the right of the decimal separator. Defaults to 2 unless `decimal-maximum-digits` is specified, in which case a number of digits between 0 and `decimal-maximum-digits` are shown. Formats other than `decimal` ignore `decimal-digits`.
+-   `decimal-maximum-digits` - when format is `decimal`, a number that controls the maximum number of digits shown to the right of the decimal separator. Possible values are from 0 to 20. Defaults to 3. If `decimal-digits` is also specified, exactly that number of digits are shown and `decimal-maximum-digits` is ignored. Formats other than `decimal` ignore `decimal-maximum-digits`.
+-   `unit-family` - when specified, displays a unit label from among a family of related units. Values are converted from a source unit (e.g. bytes) to the smallest related unit that can represent that value with magnitude >= 1 (e.g. KB, MB, etc). Set this attribute to the tag name of an element representing a family of related units, e.g. `'nimble-unit-byte'`. These unit elements are not rendered but serve to allow selective loading of data for only the needed units. The source data for the column is expected to be given in the unit specified in the tag name, e.g. for `'nimble-unit-byte'`, a source value should be a number of bytes. Supported units will be:
+    -   `'nimble-unit-byte'` - Unit labels in this family are `byte`/`bytes`, `KB`, `MB`, `GB`, `TB`, `PB`
```

Comment:
Yes, it is an elment name:
> Set this attribute to the tag name of an element representing a family of related units, e.g. `'nimble-unit-byte'`

I believe we have to take the element name directly (not an enum value) so that the client imports the tag name constant, rather than us importing all of those constants (and associated elements).